var/spack/repos/builtin/packages/squashfuse/package.py

```diff
@@ -51,6 +51,14 @@ class Squashfuse(AutotoolsPackage):
     depends_on("automake", type="build", when="@master")
     depends_on("libtool", type="build", when="@master")
 
+    def url_for_version(self, version):
+        url = "https://github.com/vasi/squashfuse/releases/download/"
+        if version > Version("0.1.104"):
```

Comment:
I think the URL change is a one-off. So:

```suggestion
        if version == Version("0.5.0"):
```